errai-bom/pom.xml

```diff
@@ -47,7 +47,7 @@
     <javaee.api.version>7.0</javaee.api.version>
     <jboss-integration-platform-bom.version>${project.parent.version}</jboss-integration-platform-bom.version>
     <!-- ip-bom uses artifact with groupId org.jboss.remoting3 -->
-    <jboss.remoting.version>4.0.0.Final</jboss.remoting.version>
+    <jboss.remoting.version>5.0.3.Final</jboss.remoting.version>
```

Comment:
@psiroky @mbiarnes This version ended up causing most of my issues. The ip-bom manages a version for jboss-remoting 3, which uses a separate package and groupId (org.jboss.remoting3). But wildfly-core depends on a newer jboss-remoting.

Should we be managing the version for org.jboss.remoting as well in the ip-bom?

Reply:
WildFly seems to be using `org.jboss.remoting:jboss-remoting:5.0.3.Final`, so I guess we should get rid of jboss-remoting3 and move the 5.0.3.Final version to ip-bom.

Reply:
@psiroky Okay. For now I will merge this because we depend on wildfly-cli directly in errai-cdi-jboss so I need to have a managed version. We can get rid of this when the ip-bom is upgraded.